Trouble downloading this action

Hey there - I'm really interested in using this action. I'm copying and pasting the job as described in the docs and seem to be getting an error while downloading the action. We are big users of github actions, but have never published one ourselves - so I don't know what is required to host the tarball of the action on github.

Here's the action that we are using

```
name: 'Check commit message style'
on:
  pull_request:
    types:
      - opened
      - edited
      - reopened
      - synchronize
  push:
    branches:
      - '*'

jobs:
  check-commit-message-style:
    name: Check commit message style
    runs-on: ubuntu-latest
    steps:
      - name: Check
        uses: mristin/opinionated-commit-message@v2
```

Here's the error:

```
Current runner version: '2.273.0'
Operating System
Virtual Environment
Prepare workflow directory
Prepare all required actions
Download action repository 'mristin/opinionated-commit-message@v2'
##[error]An action could not be found at the URI 'https://api.github.com/repos/mristin/opinionated-commit-message/tarball/v2'
```

--- a/README.md
+++ b/README.md
@@ -66,7 +66,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check
-        uses: mristin/opinionated-commit-message@v2
+        uses: mristin/opinionated-commit-message@v2.2.0
 ```
 
 ## Checked Events

